modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/rules/editor/Conditions.es.js

```diff
@@ -65,11 +65,9 @@ function FieldOperator({
 		if (operator !== '') {
 			setOperatorValue(operator);
 		}
+		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [operator]);
 
-	console.log('operator', operator);
-	console.log('operatorValue', operatorValue);
-
```

Comment:
now i see that you removed the consoles and the comments! i recommend that the next time you just do a reset of these changes and don't commit them at all, instead of undoing them in the next commit. this way the review gets easier and the code more clean :sparkles: since this PR is critical i won't ask you to fix this time, but its the ideal when we are doing commits of our code

Reply:
Hi @rebsilva. Yes, this task was made putting out the fire so I didn't have time to squash the related commits, I'm so sorry. To do it now I'll take more time and I'm afraid to break the things again. Thanks for the comprehension.

Reply:
i understand, no problem and let's keep doing our best! :heart: